openapi/components/schemas/DrsService.yaml

```diff
@@ -6,6 +6,12 @@ properties:
   maxBulkRequestLength:
     type: integer
     description: The max length the bullk request endpoints can handle (>= 1) before generating a 413 error e.g. how long can the arrays bulk_object_ids and bulk_object_access_ids be for this server.  
+  objectCount:
```

Comment:
in the documentation, these stats show as top-level in the "response sample", and in a "stats" object in the example